packages/eslint-plugin/src/configs/react.js

```diff
@@ -343,8 +343,20 @@ const getOverrides = () => [
       'import/no-webpack-loader-syntax': 'off', // this is ok in docs
     },
   },
+  {
+    files: [...configHelpers.devDependenciesFiles],
+    rules: {
+      'import/no-extraneous-dependencies': [
+        'error',
+        {
+          packageDir: [process.cwd(), configHelpers.findGitRoot()],
```

Comment:
**q:** not sure  if eslint will actively change cwd during execution? better double check

Reply:
As far as I can tell it doesn't. Some of the existing rules rely on cwd-based logic and have been working as expected, though they may need updates if we start linting the repo in one go instead of per package.